packages/safe-ds-vscode/src/extension/eda/apis/runnerApi.ts

```diff
@@ -269,6 +275,18 @@ export class RunnerApi {
             'val ' + newPlaceholderName + ' = ' + tablePlaceholder + '.getColumn("' + columnName + '").isNumeric; \n'
         );
     }
+
+    private sdsStringForRemoveColumns(columnNames: string[], tablePlaceholder: string, newPlaceholderName: string) {
+        return (
+            'val ' +
+            newPlaceholderName +
+            ' = ' +
+            tablePlaceholder +
+            '.removeColumns(["' +
+            columnNames.join('","') +
+            '"]); \n'
```

Comment:
This would break if the function gets called with an empty list for `columnNames`. It's safer to first map over the `columnNames` to enclose them in double quotes, and then join them together.